api-gateway-service/build.gradle

```diff
@@ -0,0 +1,50 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:2.1.4.RELEASE")
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'org.springframework.boot'
+apply plugin: 'io.spring.dependency-management'
+
+bootJar {
+    baseName = 'api-gateway'
+    version = applicationVersion
+    enabled = true
+}
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+dependencyManagement {
+    imports {
+        mavenBom "org.springframework.cloud:spring-cloud-dependencies:Greenwich.SR2"
```

Comment:
version should be moved to gradle properties

Reply:
we should really discuss current state of graddle in our project. I guess i've missed a lot

Reply:
this line doesn't work, so i use versions of eureka and zuul explicitly

Reply:
I meant there should be a placeholder with release train version

Reply:
But you are right We've probably missed a lot with gradle